src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Main {
+    public static final int FIGURES_NUMBER = 6;
+
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[FIGURES_NUMBER];
```

Comment:
```suggestion
        FigureSupplier figureSupplier = new FigureSupplier();
        Figure[] figures = new Figure[FIGURES_NUMBER];
```

The color must be set in FigureSupplier